src/dev/src/app/components/tree-view/tree-view-server-side-demo.ts

```diff
@@ -180,6 +202,51 @@ export class TreeViewServerSideDemo {
     }
   `);
 
+  dataExampleHtml3 = jsBeautify.html(`<fui-tree-view
+          [loading]="loading"
+          [treeNodeData]="nonRootServerSideTreeNodeData"
+          [dataRetriever]="nonRootServerSideDataRetriever"
+          [config]="{ width: '250px', height: '300px' }"></fui-tree-view>`);
+
+  dataExample3 = jsBeautify.js(`
+      // Non Root Server Side
+      nonRootServerSideTreeNodeData: NonRootTreeNode = NonRootTreeNode.instance;
+
+      nonRootServerSideDataRetriever: PagedTreeNodeDataRetriever<FoodNode> = {
+        hasChildNodes: (node: TreeNodeData<FoodNode>) => Promise.resolve(!!node.data.children && node.data.children.length > 0),
+        getChildNodeData: (node: TreeNodeData<FoodNode>) => {
+          // not used on Server Side
+        },
+        getPagedChildNodeData: (node: TreeNodeData<FoodNode>, pagingParams: PagingParams) => {
+          // On first load the initial node will be the non root tree node, dev must handle appropriately
+          const isEmptyRoot = node instanceof NonRootTreeNode;
```

Comment:
Why creating a variable if you're only using it once ? There is no need, just add you condition to the if().

Reply:
I was doing it to be more explicit for the developer to notice the difference. Its fixed now!